src/lib/commands.js

```diff
@@ -0,0 +1,271 @@
+import { TCModel, GVL } from "@iabtcf/core";
+import config from "./config";
+import {
+	encodeVendorConsentData,
+	encodeConsentData,
+} from "./cookie/cookie";
+const arrayFrom = require('core-js/library/fn/array/from');
+
+const COOKIE_VERSION = 2;
+const CMP_ID = 280;
+const CMP_VERSION = 2;
+
+const createCommands = (store) => {
+	let vendorList;
+
+	const getVendorList = (callback) => {
+		if (vendorList) {
+			callback(vendorList);
+		} else if (store.vendorList && Object.keys(store.vendorList).length) {
+			vendorList = JSON.parse(JSON.stringify(store.vendorList));
+			callback(vendorList);
+		} else {
+			const { getVendorList } = config;
+
+			if (getVendorList) {
+				getVendorList((err, data) => {
+					if (err) {
+						callback({}, err);
```

Comment:
Standard de facto states that error should be pass always as first argument to callback function and second argument should be data.

Reply:
Done, also Artur changed it in dlApi.